### PR TITLE
CHANGELOG: record past changes

### PR DESCRIPTION
**Pull request by woodruffw**
Signed-off-by: William Woodruff <william@trailofbits.com>

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -32,6 +32,18 @@ Changelog
 * Using MD5 or SHA1 in :class:`~cryptography.x509.CertificateBuilder` and
   other X.509 builders is deprecated and support will be removed in the next
   version.
+* Added additional APIs exposing
+  :class:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp` internals, including
```
Comment (alex) [inline CHANGELOG.rst:36]: Let's not describe them as internals, let's just say we exposed the new fields.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -32,6 +32,18 @@ Changelog
...
+  :attr:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp.signature_hash_algorithm`,
+  :attr:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp.signature_algorithm`,
+  :attr:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp.signature`, and
+  :attr:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp.extension_bytes`.
+* Added the :class:`~cryptography.x509.certificate_transparency.SignatureAlgorithm`
+  enum, representing valid signature algorithms per :rfc:`6962`.
```
Comment (alex) [inline CHANGELOG.rst:42]: We can skip documenting this, it's implicitly documented via  `signature_algorithm` which is the only reason to care.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -32,6 +32,18 @@ Changelog
...
+* Added :attr:`~cryptography.x509.oid.ExtendedKeyUsageOID.CERTIFICATE_TRANSPARENCY`.
```
Comment (alex) [inline CHANGELOG.rst:43]: This is too marginal to document, it can be skipped.

Comment (reaperhulk) [inline CHANGELOG.rst:43]: We don’t typically document oid additions, but otherwise this lgtm

File: CHANGELOG.rst [hunk 4]
```diff
@@ -32,6 +32,18 @@ Changelog
...
+* Added :attr:`~cryptography.x509.Certificate.tbs_precertificate_bytes`, allowing
+  users to access the to-be-signed precertificate data needed for signed
```
Comment (reaperhulk) [inline CHANGELOG.rst:45]:
```suggestion
  users to access the to-be-signed pre-certificate data needed for signed
```

**Review by alex — APPROVED** [review]

File: CHANGELOG.rst [hunk 5]
```diff
@@ -32,6 +32,15 @@ Changelog
 * Using MD5 or SHA1 in :class:`~cryptography.x509.CertificateBuilder` and
   other X.509 builders is deprecated and support will be removed in the next
   version.
+* Added additional APIs exposing
+  :class:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp` fields, including
```
Comment (alex) [inline CHANGELOG.rst:36]:
```suggestion
* Added additional APIs to
  :class:`~cryptography.x509.certificate_transparency.SignedCertificateTimestamp`, including
```

**Review by reaperhulk — APPROVED** [review]

**Review by alex — APPROVED** [review]

**Pull request merged**